mailbox/api/src/main/java/org/apache/james/mailbox/events/EventDeadLetters.java

```diff
@@ -27,7 +27,13 @@
 interface EventDeadLetters {
     Mono<Void> store(Group registeredGroup, Event failDeliveredEvent);
 
-    Flux<Event> take(Group registeredGroup);
+    Mono<Void> remove(Group registeredGroup, Event.EventId failDeliveredEventId);
 
-    Flux<Group> groups();
+    Mono<Event> failedEvent(Group registeredGroup, Event.EventId failDeliveredEventId);
+
+    Flux<Event> failedEvents(Group registeredGroup);
+
+    Flux<Event> failedEvents();
```

Comment:
This do not map any API we plan to implement and looks like it can be simplified Not to mention that we lost the group information

Reply:
Sorry, I was missreading the Webadmin apis proposal